cgcnn2/cgcnn_tr.py

```
@@ -0,0 +1,393 @@
+    else:
+        raise ValueError(
+            "Either train, valid, and test datasets or a total data directory must be provided."
+        )
+
+    # obtain target value normalizer
+    if args.task == "classification":
+        normalizer = Normalizer(torch.zeros(2))
+        normalizer.load_state_dict({"mean": 0.0, "std": 1.0})
+    else:
+        if len(total_dataset) < 500:
+            warnings.warn(
```

Ruff (0.8.2)

No explicit `stacklevel` keyword argument found

(B028)